Gemfile

```diff
@@ -82,3 +82,5 @@ end
 group :production do
   gem 'rails_12factor'
 end
+
+gem "bootstrap", "~> 5.2"
```

Comment:
We're using bootstrap from the npm package, so we should update the package, not the gem. :confused:

Reply:
Rubocop suggests single quotes. I also suggest moving this gem to the rest and putting it in alphabetical order.

Reply:
I moved the gem for the file to be sorted alphabetically. The change from the npm package to the gem is needed if we want to switch to importmaps. Switching from sprockets to importmaps will make us stop using npm as a package manager. Also importmaps handle only javascript and not css so switching to the gem is the easiest and most clean way for us to keep the css.